quiz.py

```diff
@@ -2,5 +2,51 @@
 File: quiz.py
 Author: Name
 Date: YYYY-MM-DD
-Description: A brief explanation of what this program does.
-"""
\ No newline at end of file
+Description: A Multiple choice game that calculates the score
+"""
+print("Multiple-Question Quiz Game (NBA)")
+
+correct_answers = 0
+
```

Comment:
Good use of logical paragraphs (white space). Consider using single-line comments to highlight where each question begins to make this even more clear.